cli/smartcontract/testdata/rpcbindings/invalid5/invalid.yml

```diff
@@ -6,11 +6,7 @@ events:
         type: Array
         extendedtype:
           base: Array
-          name: invalid5.NamedStruct
-namedtypes:
-  invalid5.NamedStruct:
-    base: Array
-    name: invalid5.NamedStruct
-    fields:
-      - field: SomeInt
-        base: Integer
+          name: NamedStruct
```

Comment:
Previously name contained package (`invalid5.NamedStruct`), and right now it doesn't. Why did you change this behaviour?